cdc/log.cc

```diff
@@ -836,16 +836,19 @@ class transformer final {
 
     stats::part_type_set _touched_parts;
 
-    clustering_key set_pk_columns(const partition_key& pk, api::timestamp_type ts, bytes decomposed_tuuid, int batch_no, mutation& m) const {
+    api::timestamp_type _ts;
```

Comment:
I think it's a step into a good direction but I think it would be better to introduce a new object that wraps mutation and stores _ts and _tuuid and use it inside transform. 

I think splitting the API into two function calls is not a good idea.
It is also not a good idea, in my opinion, to store state in the transformer that lives only for a length of a single function.
It's better to encapsulate _ts, _tuuid, mutation and logic around them in an object and use it inside transform.

Reply:
Now in later patch, I see that the two functions are necessary.